hack/generator/pkg/astmodel/local_package_reference_test.go

```diff
@@ -11,6 +11,10 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+func makeTestLocalPackageReference(group string, version string) LocalPackageReference {
+	return MakeLocalPackageReference("github.com/Azure/k8s-infra/hack/generated/apis", group, version)
```

Comment:
```suggestion
	return MakeLocalPackageReference("github.com/Azure/k8s-infra/hack/generated/_apis", group, version)
```

needs updating after #372

Reply:
Fixed